### DISCUSSION TITLE
When I download MOT2020 lables and 100K images

### DISCUSSION THREAD

**Discussion by ayang-1434** [bdd100k/bdd100k#235] [Q&A] [answered]
I cannot download them and will and the page will display these screens
![1651932212(1)](https://user-images.githubusercontent.com/85021361/167257781-9ffa21d5-6286-464a-8cff-f984100b2ba2.png)

**Comment by tobiasfshr** [comment, marked as answer]
duplicate of https://github.com/bdd100k/bdd100k/discussions/234
